record.go

```diff
@@ -68,15 +69,25 @@ func getRecord(host string, c Config, w http.ResponseWriter, r *http.Request) (r
 
 	r = rec.addToContext(r)
 
+	// Add the headers from record to the response
```

Comment:
Would this overwrite any headers set by the Caddyfile config?
I think we should document this here, what it overwrites. What do you think?

Reply:
It actually overwrites any header with the same key, so we should either use `Add()` method instead, or document that it overwrites the previous headers.

I guess documenting makes more sense because adding the value may cause trouble in some cases.
Or we can use the `>` for overwriting and something like `>+` for adding another value to the header.

Reply:
Happy with overwrite for now + documentation.

The semantics of overwrite would work with ">" so adding addition later on would work.